core/domain/stats_jobs_one_off.py

```diff
@@ -290,6 +293,10 @@ def prepare_reduce(cls, exp_id, values):
         # v1 fields.
         old_stats = stats_services.get_multiple_exploration_stats_by_version(
             exp_id, versions)
+        # Validate all old stats.
```

Comment:
Can you explain what's happening here? Are old stats not validated?

Maybe, thinking long-term, we should have an issue to run a one-off job to validate all of them, so that we can be confident in them?

Reply:
Also do we need to add validation for new stats coming in?